docs/RELEASE.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Conducting A Successful Release
+category: Contributing
+layout: default
+---
```

Comment:
I am familiar with this in blogs and books. What is the purpose of this header here?

Reply:
It's there for being part of the "projects github based website in the future". I was just thinking ahead here. I can remove it if you want ( or you as part of updating the documents ) and then just add it again when we have more documents to form a more complete website.

Reply:
Removed I'll just add it later again...